static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -383,3 +396,102 @@ $pillars: (
         color: #ffffff;
     }
 }
+
+
+.fc-item--pillar-special-report-alt,
+.fc-item--pillar-special-report-alt.fc-item--type-feature {
+    &.fc-item--type-immersive.fc-item--has-boosted-title.fc-item--standard-mobile {
+        .fc-item__content {
+            background-color: $special-report-alt-faded;
+        }
+
+        &:hover {
+            .fc-item__content {
+                background-color: $special-report-alt-faded;
+            }
+        }
+    }
+
+    background-color: $special-report-alt-faded !important;
```

Comment:
Regarding the abundant use of ``!important` in this PR please see discussion [here](https://github.com/guardian/frontend/pull/25602)